pkg/descheduler/descheduler.go

```diff
@@ -276,30 +277,30 @@ func Run(ctx context.Context, rs *options.DeschedulerServer) error {
 	return runFn()
 }
 
-func validateVersionCompatibility(discovery discovery.DiscoveryInterface, versionInfo version.Info) error {
-	serverVersionInfo, err := discovery.ServerVersion()
+func validateVersionCompatibility(discovery discovery.DiscoveryInterface, deschedulerVersionInfo version.Info) error {
+	kubeServerVersionInfo, err := discovery.ServerVersion()
 	if err != nil {
 		return errors.New("failed to discover Kubernetes server version")
 	}
 
-	serverVersion, err := utilversion.ParseSemantic(serverVersionInfo.String())
+	kubeServerVersion, err := utilversion.ParseSemantic(kubeServerVersionInfo.String())
 	if err != nil {
 		return errors.New("failed to parse Kubernetes server version")
```

Comment:
Could we also print `kubeServerVersionInfo.String()` in the error message? So users can see the invalid version string.